rust-peer/src/main.rs

```diff
@@ -44,7 +44,7 @@ async fn main() -> Result<()> {
     let opt = Opt::parse();
 
     let mut swarm = create_swarm()?;
-    swarm.listen_on(format!("/ip4/127.0.0.1/udp/0/webrtc").parse()?)?;
+    swarm.listen_on(format!("/ip4/0.0.0.0/udp/0/webrtc").parse()?)?;
```

Comment:
I'm curious, is this necessary to ensure the binary binds to all interfaces?

Reply:
No, but the webrtc transport should not be bound to localhost. Loopback candidates are disallowed by the ICE spec. https://www.rfc-editor.org/rfc/rfc8445#page-19